docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+| LOG_TO_TCP                  | Indicates whether logging to TCP is enabled or not (used mainly for integration tests) | No (defaults to `false`)            |
+| LOG_TO_HUB                  | Indicates whether logging to IoT Hub is enabled or not                                 | No (defaults to `false`)            |
+| LOG_TO_TCP_ADDRESS          | IP address for TCP logs                                                                | Yes, only if TCP logging is enabled |
+| LOG_TO_TCP_PORT             | TCP port to send logs to                                                               | Yes, only if TCP logging is enabled |
+| NETID                       | Gateway network ID                                                                     | No (defaults to network id 1)       |
+| AllowedDevAddresses         | Allowed Dev addresses                                                                  | No                                  |
+| LNS_SERVER_PFX_PATH         | Path of the .pfx certificate to be used for LNS Server endpoint                        | No                                  |
+| LNS_SERVER_PFX_PASSWORD     | Password of the .pfx certificate to be used for LNS Server endpoint                    | No                                  |
+| CLIENT_CERTIFICATE_MODE     | Specifices the client certificate mode with which the server should be run             | No (defaults to `NoCertificate`)    |
+| LNS_VERSION                 | Version of the LNS                                                                     | No                                  |
+| IOTHUB_CONNECTION_POOL_SIZE | IoT Hub connection pool size                                                           | No (defaults to 1)                  |
+
+Additional settings are available via desired properties of the Network Server module twin in IoT Hub:
+
+| Property name                 | Description                                                                                                                                   | Mandatory               |
+| ----------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------- |
+| FacadeServerUrl               | Azure Facade function url                                                                                                                     | Yes                     |
+| FacadeServerAuthCode          | Azure Facade function auth code                                                                                                               | Yes                     |
+| ProcessingDelayInMilliseconds | Processing delay (in milliseconds) which should be used for LNS that does not own connection for a device (applicable to multi-gateway setup) | No (defaults to 400 ms) |
```

Comment:
Let's change the description:
Processing delay (in milliseconds) to be used for the LNS not owning the connection for a device in a multi-gateway scenario (with link to scalability document)